README_DEV.md

```diff
@@ -1,16 +1,38 @@
-## Development
-### Android Studio
+# Development
+## Android Studio
 This project was developed using Android Studio Electric Eel, but you can also use Android Studio Dolphin since it's currently using Android Gradle Plugin version 7.3.1.
 
-### How to run the unit tests
+## Get up and running locally
```

Comment:
can we mention the following steps for MacOS?

---

### On macOS
```bash
brew install openjdk@17
```

_Alternatively, you can install a [JDK from the Oracle website](https://www.oracle.com/java/technologies/downloads/#java17)_

....


this way, the developer can skip the path step (as homebrew will handle this^)

Reply:
addressed (hopefully) in [18d3919](https://github.com/teamforage/forage-android-sdk/pull/44/commits/18d3919bd5725007504ca687fca69e44387aa999)